### PR TITLE
rust-geom API changes

### PR DESCRIPTION
**Pull request by frewsxcv**
https://github.com/servo/rust-geom/pull/81

### CODE REVIEW

**Comment by mbrubeck** [comment]
r+ for this commit, but should also bump the geom version in Cargo.toml if you bump it in servo/rust-geom#81

**Comment by frewsxcv** [comment]
> but should also bump the geom version in Cargo.toml 

The version isn't specified in this project's Cargo.toml; it just has a git path specified

**Comment by mbrubeck** [comment]
Oh!  I guess we should get this stuff onto crates.io.  But we can do that separately.

This is good to land whenever ready.

**Comment by frewsxcv** [comment]
I need to finish making changes to servo and maybe other libraries before any of this lands. Was planning on finishing this stuff later tonight

**Comment by frewsxcv** [comment]
> I guess we should get this stuff onto crates.io. But we can do that separately.

Is it still worth bumping rust-geom to 0.2.0 or should I just save that for a later PR

**Comment by mbrubeck** [comment]
> Is it still worth bumping rust-geom to 0.2.0 or should I just save that for a later PR

I guess there's not much point if it's not on crates.io, so you can ignore that for now.

**Comment by frewsxcv** [comment]
servo/rust-geom#81 has merged. Just force pushed and tests pass. This is ready for review now.

File: src/tiling.rs [hunk 1]
```diff
@@ -148,9 +148,9 @@ impl TileGrid {
 
         // NB: Even in the case of an empty rect, the semantics of Rust floating-point-to-integer
         // casts mean this will corrently round to zero.
-        (Point2D((rect.origin.x / self.tile_size.get() as f32) as usize,
+        (Point2D::new((rect.origin.x / self.tile_size.get() as f32) as usize,
                  (rect.origin.y / self.tile_size.get() as f32) as usize),
-         Point2D(((rect.origin.x + rect.size.width - 1.0) / self.tile_size.get() as f32) as usize,
+         Point2D::new(((rect.origin.x + rect.size.width - 1.0) / self.tile_size.get() as f32) as usize,
                  ((rect.origin.y + rect.size.height - 1.0) / self.tile_size.get() as f32) as usize))
```
Comment (Ms2ger) [inline src/tiling.rs]: Indentation here and in the next hunk

Comment (frewsxcv) [inline src/tiling.rs]: https://github.com/frewsxcv/rust-layers/commit/08bb552c45121bf8b932798003101f486570fa6f

**Pull request merged**